libqtile/backend/wayland/inputs.py

```diff
@@ -142,6 +143,8 @@ class InputConfig(configurable.Configurable):
         ("kb_variant", None, "Keyboard variant i.e. ``XKB_DEFAULT_VARIANT``"),
         ("kb_repeat_rate", 25, "Keyboard key repeats made per second"),
         ("kb_repeat_delay", 600, "Keyboard delay in milliseconds before repeating"),
+        ("kb_numlock", None, "``True`` or ``False`` Enable numlock on startup"),
+        ("kb_capslock", None, "``True`` or ``False`` Enable capslock on startup"),
```

Comment:
Perhaps default these to `False`?

Reply:
I was following the doc string hence the `None` as default value, I don't mind changing to `False` as it would remove the type hinting error.

Reply:
Oh good point. A device might (for some reason) set this itself, and using `None` lets the device have control rather than us forcing it either way. I don't know if devices actually do that in practice, but let's leave it just in case.